### PR TITLE
feat: added documentation for migrating from an old Enterprise server to the new Docker servers

### PR DESCRIPTION
**Pull request by bcoe**
CC: @nexdrew @ashleygwilliams

### CODE REVIEW

File: content/enterprise/migrating.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<!--
+title: Migrating from Old Servers
+-->
+
+# migrating from pre-docker npm enterprise installations
```
Comment (ashleygwilliams) [inline content/enterprise/migrating.md]: please use capitalization :)

File: content/enterprise/migrating.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+2. ensure that the `Full URL of npm Enterprise registry` is set to the appropriate
+   full-URL of the registry (should match the old server).
+3. switch your DNS entries so that they are pointing at the new Enterprise server.
+4. smoke test the new server and retire the old server.
+
+## if you would like to use a new DNS name
+
+1. start the new server.
+2. you will need to run [this migration tool](https://www.npmjs.com/package/rewrite-url-follower).
+  * the command you run will look like this: `rewrite-url-follower rewrite --rewrite=http://my-new-server-host:8080 --host=172.17.0.1 --port=5984`.
```
Comment (nexdrew) [inline content/enterprise/migrating.md]: Lots of `rewrite`s going on here. :)

Comment (bcoe) [inline content/enterprise/migrating.md]: I'd be tempted to hold off on this documentation until I finish the proxy that makes rewriting a thing of the past.

**Comment by ashleygwilliams** [comment]
update @bcoe ?

**Comment by bcoe** [comment]
@ashleygwilliams I'm hoping to release something this week that would allow us to eliminate several steps from the migration process, why don't I update this pull as soon as that's out the door?

**Comment by ashleygwilliams** [comment]
closing as we have npme docs now!

**Pull request closed without merging**